javafx-archetype-simple/src/main/resources/archetype-resources/pom.xml

```diff
@@ -6,24 +6,26 @@
     <version>${version}</version>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+#if ("8" == ${javafx-version})
```

Comment:
this might work only if user sets `javafx-version` to 8, but what if he/she uses `1.8`?

Reply:
Shouldn't these if..else statements also be a part of FXML archetype?